pyseir/inference/model_fitter.py

```diff
@@ -680,6 +680,40 @@ def run_for_fips(cls, fips, n_retries=3):
         return model_fitter
 
 
+def _execute_model_for_fips(fips):
```

Comment:
When are there not fips for this? I would rather handle the missing fips in the calling function if possible

Reply:
There is some weird fips cascading logic where sometimes some counties don't show up.  similiar to the other case.

Reply:
Are you sure this isn't because of the leading zeros issue?

Reply:
yes